components/Heroes/HeroesIndex.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { REDUCER_KEY } from './../../reducers';
+import HeroList from './HeroList';
+import style from './heroes.css';
+
+const HeroesIndex = ({ heroes }) => (
+  <div>
+    <h1 className={style.Header}>All Dota 2 Heroes</h1>
+    <HeroList heroes={heroes} />
+  </div>
+);
+
+const mapStateToProps = (state) => ({
+  heroes: state[REDUCER_KEY].heroes,
+});
```

Comment:
I'd rather use the 'selector' syntax (doesn't need to be a reselect selector) in future components. So for example, instead of coupling your components to the shape of the state, have an exported object of functions which retrieve specific parts of the state base and accept state as a parameter. Look at /reducers/gotPlayer/matches.js for some examples.